src/datajudge/db_access.py

```diff
@@ -904,77 +904,149 @@ def get_column_array_agg(
     return result, selections
 
 
+def _cdf_selection(engine, ref: DataReference, cdf_label: str, value_label: str):
+    col = ref.get_column(engine)
+    selection = ref.get_selection(engine).subquery()
+
+    # Step 1: Calculate the CDF over the value column.
```

Comment:
Just curious: Is possible to merge the two steps? Like so

```py
sa.select([
  selection.c[col],
  sa.func.max(sa.func.cume_dist().over(order_by=col)),
])
.group_by(selection.c[col])
```

Reply:
I wondered the same and doing it that way leads to an error:
`sqlalchemy.exc.ProgrammingError: (psycopg2.errors.GroupingError) aggregate function calls cannot contain window function calls`

Reply:
Could you add in the docstring a bit more information about the objective/idea behind this method? 

It's great to have the comments on the step-by-step like in the SQL version before, but a summary would be a great addition to it, particularly clarifying and being explicit about the meaning of the arguments.

Reply:
Good idea.
[f396053](https://github.com/Quantco/datajudge/pull/44/commits/f3960539612780cfa26aea64e508d6e48bde9b32)